package.json

```diff
@@ -37,7 +37,7 @@
         "@guardian/ab-react": "1.0.0-next.0",
         "@guardian/atoms-rendering": "^1.5.0",
         "@guardian/automat-client": "^0.2.16",
-        "@guardian/consent-management-platform": "4.0.0-9",
+        "@guardian/consent-management-platform": "4.0.0-13",
```

Comment:
This is currently a pre-release; we will switch to a major release when that it put out.